ui/src/Components/SearchCandidates.jsx

```diff
@@ -133,16 +168,22 @@ export default class SearchCandidates extends React.Component {
 							</Col>
 						</FormGroup>
 						<FormGroup row>
-							<Label for="phone" sm={2}>Phone Number</Label>
+							<Label for="verified" sm={2}>Verified</Label>
+							<Col sm={10}>
+							<input type="checkbox" className="font-small d-flex align-items-center" onChange={this.onVerifiedSearch}/>
+							</Col>
+						</FormGroup>
+						<FormGroup row >
```

Comment:
![image](https://user-images.githubusercontent.com/37027135/101666303-e61df280-3a02-11eb-919f-114106dba4a6.png)
This looks off. The switches move when clicked and has wonky behaviour

Reply:
![image](https://user-images.githubusercontent.com/37027135/101666386-0188fd80-3a03-11eb-81da-e7d0f050a835.png)
![image](https://user-images.githubusercontent.com/37027135/101666404-06e64800-3a03-11eb-8d24-d4fe0a6d65ae.png)